src/main/java/org/example/Main.java

```diff
@@ -50,35 +41,55 @@ public static void main(String[] args) throws Exception {
             JsonObject jsonObject = new Gson().fromJson(response.toString(), JsonObject.class);
             JsonArray results = jsonObject.getAsJsonArray("results");
 
+            Class.forName("org.postgresql.Driver");
+
             Connection jDbCconnection = DriverManager.getConnection("jdbc:postgresql://localhost:5432/dex");
-            PreparedStatement statement = jDbCconnection.prepareStatement("INSERT INTO medication_test (brand_name) VALUES (?)");
+            PreparedStatement statement = jDbCconnection.prepareStatement("INSERT INTO medication_test2 (brand_name, strength, dosage_form) VALUES (?,?,?)");
 
```

Comment:
I created a new table to test how it would look now that i figured out how to unwrap the results

Reply:
For line 47 what is the meaning of (?) in VALUES?